charts/csi-baremetal-deployment/values.yaml

```diff
@@ -50,6 +52,10 @@ driver:
       requests:
         cpu:
         memory:
+    securityContext:
```

Comment:
Maybe it's better to have this as default values? I mean denote it in the CRDs

Reply:
Added default values for SecurityContext fields